pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
-    <version>68.0.0.247</version>
+    <version>70.0.0.1287</version>
```

Comment:
This is an unreleased version of the parent. We should depend on today's 71.0.0 release.

Reply:
Fixed.